src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,200 @@
+    it: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    fr: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+
+
+class OpenDataSwissOperationTimeDaysOfWeek(Enum):
+    MONDAY = 'MONDAY'
+    TUESDAY = 'TUESDAY'
+    WEDNESDAY = 'WEDNESDAY'
+    THURSDAY = 'THURSDAY'
+    FRIDAY = 'FRIDAY'
+    SATURDAY = 'SATURDAY'
+    SUNDAY = 'SUNDAY'
+
+    def to_osm_opening_day_format(self) -> str:
+        return {
+            self.MONDAY: 'Mo',
+            self.TUESDAY: 'Tu',
+            self.WEDNESDAY: 'We',
+            self.THURSDAY: 'Th',
+            self.FRIDAY: 'Fr',
+            self.SATURDAY: 'Sa',
+            self.SUNDAY: 'Su',
+        }.get(self, None)
+
+
+@validataclass
+class OpenDataSwissOperationTimeInput:
+    operatingFrom: Optional[str] = Noneable(StringValidator())
+    operatingTo: Optional[str] = Noneable(StringValidator())
+    daysOfWeek: Optional[list[str]] = Noneable(ListValidator(EnumValidator(OpenDataSwissOperationTimeDaysOfWeek)))
+
+
+@validataclass
+class OpenDataSwissPropertiesInput:
+    operator: str = StringValidator()
+    displayName: str = StringValidator()
+    address: Optional[OpenDataSwissAddressInput] = Noneable(DataclassValidator(OpenDataSwissAddressInput))
+    capacities: list[OpenDataSwissCapacitiesInput] = ListValidator(DataclassValidator(OpenDataSwissCapacitiesInput))
+    additionalInformationForCustomers: Optional[OpenDataSwissAdditionalInformationInput] = Noneable(
+        DataclassValidator(OpenDataSwissAdditionalInformationInput)
+    )
+    parkingFacilityCategory: str = StringValidator()
+    parkingFacilityType: Optional[str] = Noneable(StringValidator())
+    salesChannels: Optional[list[str]] = Noneable(ListValidator(StringValidator()))
+    operationTime: Optional[OpenDataSwissOperationTimeInput] = Noneable(DataclassValidator(OpenDataSwissOperationTimeInput))
+
+    def __post_init__(self):
+        for capacity in self.capacities:
+            if capacity.categoryType == OpenDataSwissCapacityCategoryTypeInput.STANDARD:
+                return
+        # If no capacity with type PARKING was found, we miss the capacity and therefore throw a validation error
+        raise ValidationError(reason='Missing parking spaces capacity')
+
+    def get_osm_opening_hours(self) -> str:
+        open_swiss_opening_times_by_weekday: dict[OpenDataSwissOperationTimeDaysOfWeek, list[str]] = defaultdict(list)
+        check_counter_24_7: int = 0
+        check_list_weekday: list[str] = []
+
+        # OSM 24/7 has no secs in its timeformat and no endtime 00:00, so we replace with 24:00 and remove the secs
+        closing_hh_mm_ss: list[str] = list(self.operationTime.operatingTo.replace('00:00:00', '24:00:00').split(':'))
+        opening_hh_mm_ss: list[str] = list(self.operationTime.operatingFrom.split(':'))
+        opening_time: str = f'{opening_hh_mm_ss[0]}:{opening_hh_mm_ss[1]}-{closing_hh_mm_ss[0]}:{closing_hh_mm_ss[1]}'
```

Comment:
You do not validate the time format before, so `opening_hh_mm_ss[1]` can run into an exception. I would suggest to use a `TimeValidator` in order to catch these issues right at the start.